packages/vscode-extension/package.json

```diff
@@ -7,7 +7,8 @@
   "version": "2.0.0-next.4",
   "publisher": "Neo4j Inc.",
   "activationEvents": [
-    "onLanguage:cypher"
+    "onLanguage:javascript",
+    "onLanguage:typescript"
```

Comment:
I think this is not from this pr right? Looks more like you were trying how to provide embedded support for Cypher?

Reply:
yes that's right